packages/documentation-site/patternfly-docs/content/accessibility/skip-to-content/skip-to-content.md

```diff
@@ -3,12 +3,46 @@ id: Skip to content
 section: components
 ---
 
-A **skip to content** is a component that allows screen reader and keyboard users to bypass navigation rather than tabbing through it.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Keyboard users** should be able to find the skip to content using **Tab** at the start of a page with navigation. They should be able to select the skip to content using **Enter**. A skip to content does not trap tab focus. It is the first tabbable element on a page, but can be tabbed past and into the navigation of a page if the user prefers.
+## Accessibility
 
-**Screen reader users** should also be able to navigate to skip to content using **Tab** and select it to skip past the navigation of a page.
+To implement an accessible PatternFly **skip to content**:
 
-## To make skip to content accessible:
+- Render the skip to content as the first focusable element on the page.
 - Ensure that the skip to content href is properly linked to the main content of the page. You may have to add context to the href if there are any query parameters in the URL.
-- If you're using the page component with the skip to content, we add a tabindex="-1" for you. If you are using the skip to content on its own, give the target of the skip to content link a tabindex of -1. Some browsers require this attribute to make this component work consistently.
+
+## Testing
+
+At a minimum, a skip to content should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="skip-to-content-a11y-checkbox-1" label="The skip to content is the first focusable element on the page." description="When content is repeated on several pages - such as actions in a header or navigation links in a sidebar - users navigating via keyboard or other assistive technologies would need to navigate through that content every time. Placing a skip to content as the first focusable element helps users avoid this repeated content." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="skip-to-content-a11y-checkbox-2" label={<span>The skip to content's <code className="ws-code">href</code> is linked to the main content of the page.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="skip-to-content-a11y-checkbox-3" label="Standard keyboard navigation can be used to navigate between the skip to content and other focusable elements." description={<span><kbd>Tab</kbd> navigates to the skip to content when at the beginning of a page or the next focusable element thereafter, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element or the skip to content.</span>} />
```

Comment:
I find this description just a tad confusing. I previously tried to explain what to expect with the keyboard interactions surrounding the skip to content, that it was essentially still in the tab order and that users could interact with it or move past it/that it doesn't force the user into using it. I think the part that I find particularly confusing is this line, "Shift Tab navigates to the previous focusable element or the skip to content." For some reason this feels odd, like we're just describing general keyboard interactions?

Reply:
That's a fair point, yeah this is basically the keyboard navigation content that is used in other components' a11y docs. It does end up a bit wordy because it's sort of a special case in that the first Tab on a page should focus the Skip to Content.

Do you think this item is necessary, or should we just omit the Shift + Tab part? Could also reword it a little, though not sure if this really improves it significantly:

```
Depending on where focus is on the page, <kbd>Tab</kbd> navigates to the skip to content or the next focusable element, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element or the skip to content. Since the skip to content should be the first focusable element on a page, the previous focusable element may typically be an item in the browser toolbar.
```

Basically trying to convey that Tab and Shift + Tab work and what the expected focus should be.

Reply:
Yeah, I think the edit makes it a lot clearer. It adds context so I better understand what you're saying. I'm good with that, but if you think it's better to remove it altogether, that's fine with me too.